### DISCUSSION TITLE
Steam Mobile confirmation when delete or gift items.

### DISCUSSION THREAD

**Discussion by matr9xnb** [ValveSoftware/Dota2-Gameplay#20252] [Ideas]
I think we need to add a feature that allows turning on or off confirmation mode on the Steam Mobile app when deleting or gifting DotA items while in-game Armory. This is the same feature when we trade or sell items on the market.

**Comment by jerry4422cherry** [comment]
That sounds like a great idea! Adding a confirmation mode for deleting or gifting Dota items in the Steam Mobile app would definitely help prevent accidental actions, especially since these items can be valuable to players. It would enhance user control and provide peace of mind.
[ADP Vantage HCM](https://www.adpvantages.com)
You might want to suggest this feature directly through Steam’s feedback channels, where they often review user suggestions for future updates. It’s always great to see players advocating for improvements!
